frontend/app/components/module/depobio.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component } from '@angular/core';
+import { AuthService, User } from '@geonature/components/auth/auth.service';
+import { DemarcheSimplifieeService } from '../../services/demarche-simplifiee.service';
+import { FolderData } from './interfaces';
+
+@Component({
+  selector: 'app-depobio',
+  templateUrl: './depobio.component.html',
+  styleUrls: ['./depobio.component.css'],
+})
+export class DepobioComponent {
+  folderNumber: string = '';
+  message: string = '';
+  isError: boolean = false;
+  isLoading: boolean = false;
+  folderData: FolderData = null;
```

Comment:
```suggestion
  fileData: FileData | null = null;
```

Erreur : `Impossible d'assigner le type 'null' au type 'FolderData'`.